src/github.js

```diff
@@ -29,17 +34,21 @@ module.exports.shouldProcess = function (data) {
   }
 
   var title = data.pull_request.title
+  var body = data.pull_request.body
   var change = data.changes && data.changes.title && data.changes.title.from ? data.changes.title.from : null
-  var titleId = match(title)
+  var id = match(title)
+  if (!id) {
+    id = match(body)
+  }
```

Comment:
Lines 36 - 42 (excluding 38) can be replaced with `module.exports.getAsanaId`

Reply:
or `exports.getAsanaId` maybe?

Reply:
done